contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -2912,7 +2912,12 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 
 					if (islogin)
 					{
-						if (!has_privs_of_role(GetSessionUserId(), get_role_oid("sysadmin", false)))
+						/*
+						 * Check if the current login has privileges to create
+						 * login.
+						 */
+						if (!has_privs_of_role(GetSessionUserId(), get_sysadmin_oid()) &&
+								!has_privs_of_role(GetSessionUserId(), get_securityadmin_oid()))
```

Comment:
Alter login should not work if current user is member of security admin and the login being altered is member of sysadmin

Reply:
Please validate and add test for the same as well.